ethereum/spec/src/main/java/tech/pegasys/teku/spec/config/Constants.java

```diff
@@ -66,8 +66,8 @@ public class Constants {
   public static final int PROTOARRAY_FORKCHOICE_PRUNE_THRESHOLD = 256;
 
   // Teku Sync
-  public static final UInt64 MAX_BLOCK_BY_RANGE_REQUEST_SIZE = UInt64.valueOf(200);
-  public static final UInt64 SYNC_BATCH_SIZE = UInt64.valueOf(50);
+  public static final UInt64 HISTORICAL_SYNC_BATCH_SIZE = UInt64.valueOf(50);
```

Comment:
Because no one realistically uses single peer sync. It is disabled by default with a hidden option so made the batch size same as the multipeer sync to avoid any issues when moving to Deneb and for consistency.

Reply:
Good to know, thanks